tests/lib/rules/fluent-chaining.invalid.js

```diff
@@ -0,0 +1,96 @@
+// function body should be on separate lines
+a()
+   .b(function() { return 'b-invalid'; });
```

Comment:
Let's pull this "function body should be on separate lines" part out of `fluent-chaining`. If we were going to make that a rule, it really should be a rule of its own (similar to the "Function calls must begin and end with the same indentation" rule mentioned above). That said, this rule isn't actually part of our styleguide, so should just be removed altogether. We do allow one-liners for situations such as the following:

```
somePromiseThatReturnsAnUnneededValue()
   .then(function() { done(); })
   .catch(done);
```

In that case, you can't just have `.then(done)` because some frameworks (e.g. mocha) will take anything that's passed to `done` and treat it like there was an error, etc.

Or cases like this make one-liners handy:

```
return _.chain(values)
   .filter(function(v) { return v >= 10; })
   .somethingElse()
   .value();
```